public/app/features/logs/components/LogRows.tsx

```diff
@@ -63,6 +64,7 @@ export interface Props extends Themeable2 {
   scrollIntoView?: (element: HTMLElement) => void;
   isFilterLabelActive?: (key: string, value: string, refId?: string) => Promise<boolean>;
   pinnedRowId?: string;
+  pinnedLogs?: ContentOutlineItemContextProps[];
```

Comment:
Can we change that type to a string array of IDs to avoid importing the Explore type `ContentOutlineItemContextProps` here?